### PR TITLE
chore: migrate mvc samples vol5 telerik/kendo#20556

### PR DESCRIPTION
**Pull request by eyupyusein**

### CODE REVIEW

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/Bundles/Views/_ViewStart.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/Bundles/Views/_ViewStart.cshtml:1]: The example should show how to use the kendo ui js and CSS files in bundles. Add a BundleConfig file and add the necessary kendo bundles in it: https://github.com/telerik/ui-for-aspnet-mvc-examples/blob/master/general/KendoAspNetBundles/KendoAspNetBundles/App_Start/BundleConfig.cs
Then use the Area's own Layout to render the bundles: https://github.com/telerik/ui-for-aspnet-mvc-examples/blob/master/general/KendoAspNetBundles/KendoAspNetBundles/Views/Shared/_Layout.cshtml

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/Captcha/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/Captcha/Views/Home/Index.cshtml:1]: The Captcha's audio does not work.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ChartSignalR/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ChartSignalR/Views/Home/Index.cshtml:1]: There should be an iframe at the bottom of the Index page in which a Grid declared in different view should be displayed: 
```
<iframe src="@Url.Action("Grid")" style="width: 100%; height: 400px;" frameborder="0"></iframe>
```
https://github.com/telerik/ui-for-aspnet-mvc-examples/blob/master/chart/signalr-bound-chart/KendoUIMVC5/Views/Home/Grid.cshtml

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/GridEditingWebApi/Models/ProductsController.cs
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/GridEditingWebApi/Models/ProductsController.cs:1]: The ProductsController.cs file should be in the Area's Controllers folder.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ODataWebApiWidgets/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ODataWebApiWidgets/Views/Home/Index.cshtml:1]: Attempting to update a record results in a failing request: 
Failed to load resource: the server responded with a status of 400 ()

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ODataWebApiWrappers/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/ODataWebApiWrappers/Views/Home/Index.cshtml:1]: The Editable configuration of the second Grid is not set and on attempting to edit a record nothing happens.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerEditingWebApi/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerEditingWebApi/Views/Home/Index.cshtml:1]: The CRUD operations do not work:
PUT https://localhost:44346/SchedulerEditingWebApi/Home/api/Task/25 404 (Not Found)

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerGoogleAPI/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerGoogleAPI/Views/Home/Index.cshtml:1]: The Scheduler is not loaded with data:
Failed to load resource: the server responded with a status of 400 ()

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerSignalRLocalHub/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerSignalRLocalHub/Views/Home/Index.cshtml:1]: No data is loaded in the Scheduler. If you attempt to create a new event, an error is thrown:
Uncaught TypeError: Cannot read properties of undefined (reading 'server')

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerSignalRServerFiltering/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerSignalRServerFiltering/Views/Home/Index.cshtml:1]: A js exception on loading the page: 
Uncaught Error: The "promise" option must be a Promise.
The Scheduler is non-functional.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerValidatingTimeslot/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerValidatingTimeslot/Views/Home/Index.cshtml:1]: The Read request fails:
Failed to load resource: the server responded with a status of 500 ()

The message for unavailable slots appears when trying to add an event on slots that should be available.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/UploadAmazonS3/Views/Home/Index.cshtml
Comment (IvanDanchev) [inline Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/UploadAmazonS3/Views/Home/Index.cshtml:1]: Remove all the unnecessary tags like **html** and **body** from the view, as well as the old CSS and js references. Only the Upload should be present in the view.

**Review by IvanDanchev — CHANGES_REQUESTED** [review]
Comments left for issues found in the individual examples

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerGoogleAPI/Views/Home/Index.cshtml
Comment (eyupyusein) [reply Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerGoogleAPI/Views/Home/Index.cshtml:1]: The sample expects the user to enter his own api key: 
...calendar.google.com/events?key=YourAPI_KEY

**Comment by eyupyusein** [comment]
All errors have been resolved.

File: Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerValidatingTimeslot/Views/Home/Index.cshtml
Comment (IvanDanchev) [reply Telerik.Examples.Mvc/Telerik.Examples.Mvc/Areas/SchedulerValidatingTimeslot/Views/Home/Index.cshtml:1]: After the change, the following exception is thrown on loading the page when the Scheduler sends a request to the Meetings_Read action:
**System.Data.SqlClient.SqlException: Invalid object name 'dbo.MeetingAtendees'.**

**Review by IvanDanchev — APPROVED** [review]
All looks good after the latest commit

**Pull request merged**